### PR TITLE
Added policy_server:: exclusion to avoid nodes pulling code from github

### PR DESCRIPTION
**Pull request by Enemico**

### CODE REVIEW

**Comment by Enemico** [comment]
@nickanderson this was left behind, but now i work here!

**Comment by nickanderson** [comment]
+1

**Comment by Enemico** [comment]
@nickanderson i added some more changes here: once the hub is bootstrapped, you want to push your pristine /masterfiles directory to github in order to start writing policies on your local repo, and push them to github and subsequently automatically having them pulled further to the hub.

**Comment by Enemico** [comment]
@nickanderson sorry, i forgot to add that is necessary to put the timestamp and release id into a .gitignore, and is quite important if one want this setup to be functional without headaches. So here it is.

**Comment by Enemico** [comment]
@nickanderson added a short sentence about adding the new policy to promises.cf (probably redundant).

File: guide/installation-and-configuration/version-control.markdown [hunk 1]
```diff
@@ -46,9 +70,24 @@ potentially bad merges. This assumes that no development takes place in
       chdir => "/var/cfengine/masterfiles";
     }
 ```
+This policy will then regularly and periodically pulling the masterfiles
```
Comment (kacfengine) [inline guide/installation-and-configuration/version-control.markdown]: Minor detail: I vaguely recall that there is a bug which either misrenders or fails to render the page correctly if there is no blank line between the text and example. I might be misremembering though. You don't need to fix it now, but keep an eye on the documentation after this is merged.

**Comment by kacfengine** [comment]
Good stuff!

**Comment by kacfengine** [comment]
I'll cherry-pick to 3.7 once I see the page renders correctly.

File: guide/installation-and-configuration/version-control.markdown (hunk 1 above)
Comment (kacfengine) [inline guide/installation-and-configuration/version-control.markdown]: Nope, I'm tripping, there is no bug here! :-)

**Comment by kacfengine** [comment]
Cherry-picked to 3.7.

**Pull request merged**